src/component/task_list.rs

```diff
@@ -59,12 +56,17 @@ impl TaskList {
                 "K",
                 "Moves the task up on the task list",
             ),
+            HelpAction::new(
+                KeyCode::Char('h'),
+                "h",
+                "Gives selected task lower priority",
+            ),
             HelpAction::new(KeyCode::Char('t'), "t", "Add tags to the task"),
-            HelpAction::new(KeyCode::Char('s'), "s", "Toggle task sort by priority"),
+            HelpAction::new(KeyCode::Char('s'), "s", "Toggle task sort (by priority)"),
```

Comment:
Just realised, this should probably just be `Sort tasks`

Reply:
Oh, good point)

Reply:
How about this?
```rust
HelpAction::new(KeyCode::Char('t'), "t", "Adds tags to the task"),
HelpAction::new(KeyCode::Char('s'), "s", "Sorts tasks (by priority)"),
HelpAction::new(
    KeyCode::Char('S'),
    "S",
    "Toggles automatic task sort",
),
```

Reply:
Sounds good!